src/_tutorials/server/cmdline.md

```diff
@@ -91,7 +91,7 @@ brief description of key app features, click the highlighted code below.
 
 {% include_relative _dcat-example.html %}
 
-Here's an example of running the app from the command line:
+You can run the app from the command line over the quote.txt file (which you can download from this [repo] ({{site.repo.this}}/blob/master/examples/misc/test_data/quote.txt?raw=1){:target="_blank"}) or any other text file, as shown here:
```

Comment:
We try to keep lines 80 characters or less. (Not possible here, perhaps, but still, the lines could be shorter.

Also, how about we make the link more clearly to the quote.txt file? Also, make sure there's no space between the `]` and the `(`. Maybe:

```
You can run the app from the command line over `quote.txt`
([downloadable file]({{site.repo.this}}/blob/master/examples/misc/test_data/quote.txt?raw=1){:target="_blank"})
or any other text file:
```